cmd/vic-machine/create/create.go

```diff
@@ -239,6 +241,12 @@ func (c *Create) Flags() []cli.Flag {
 			Usage:       "Virtual Container Host x509 certificate file",
 			Destination: &c.cert,
 		},
+		cli.StringFlag{
+			Name:        "base-image-size",
```

Comment:
@fdawg4l I thought it's the image datastore size.
Is there any data in the scratch disk? I'm just not clear what does this mean for end user